docs/post-install.md

```diff
@@ -61,9 +58,9 @@ sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keyc
 
 ### Activate Drone
 
-[Gitea](https://gitea.io/en-us/) and [Drone](https://www.drone.io/) are an integral part of how Otomi cluster configurations are stored and updated. 
+[Gitea](https://gitea.io/en-us/) and [Drone](https://www.drone.io/) are an integral part of how Otomi cluster configurations are stored and updated.
 
-1. Click on the **Gitea** app (under Platform/Otomi Apps) in the console. 
+1. Click on the **Gitea** app (under Platform/Otomi Apps) in the console.
```

Comment:
why is Gitea referenced here? It will simply pop up when drone is started. It could be github or whatever oauth app that is configured. Gitea has no special role here.